client/components/HouseRules.jsx

```diff
@@ -1,9 +1,60 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const HouseRules = () => (
-  <div className="HouseRules">
-    HouseRules
+const HouseRules = ({ rules, showHouseRules, toggleHouseRules }) => (
+  <div className="house-rules">
+    <h3>
+      House Rules
+    </h3>
+    <div>
+      {console.log(rules)}
+      {Object.keys(rules).map(
+        rule => (rules[rule] && rule !== 'id' && rule !== 'Custom message' ? (
+          <div>
+            {rules[rule]}
+          </div>
+        )
+          : ''),
+      )}
+    </div>
+    <div className={`read-all-rules-${showHouseRules}`}>
+      {Object.keys(rules).filter(rule => rule === 'Custom message')}
+    </div>
+    <span role="button" tabIndex="0" className="read-more" onClick={() => { toggleHouseRules(); }} onKeyPress={() => { toggleHouseRules(); }}>
+      {
+        !showHouseRules
+          ? (
+            <span className="bold">
+              Read all rules
+            </span>)
+          : (
+            <span className="bold">
+              Hide ^
+            </span>
+          )
+      }
+    </span>
+    <div className="divider" />
```

Comment:
You can try using css border to create a bottom divider line! 
Just saying there's different way to do it.